test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/DelegatingTestExporter.cs

```diff
@@ -0,0 +1,43 @@
+// <copyright file="DelegatingTestExporter.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+
+namespace OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests
+{
+    public class DelegatingTestExporter<T> : BaseExporter<T>
```

Comment:
Might be useful to have this in `$\test\OpenTelemetry.Tests\Shared\` so other test projects could use it if needed.